src/nfa/operations.cc

```diff
@@ -874,6 +905,8 @@ std::ostream& std::operator<<(std::ostream& os, const Mata::Nfa::Nfa& nfa) {
     return os;
 }
 
+//Are the following functions for alphabets there to stay or to be refacvtored together with alphabets?
```

Comment:
They are meant to stay (we use them in Noodler), but they will work with different alphabets and they may be refactored due to that quite a bit.

Reply:
ok. Maybe we would not need all the variants?

Reply:
We do not know yet. We will have to refactor alphabets, refactor how we work with alphabets in Noodler, and finally review which of these functions we need and use and which can be deleted.